### PR TITLE
Kamil

### PR DESCRIPTION
**Pull request by kusta4292**
AC-1 and AC-1 is done

### CODE REVIEW

File: src/test/java/com/project/step_definitions/DeletedFilesTab_StepDefinitions.java
Comment (rokudaimme) [inline src/test/java/com/project/step_definitions/DeletedFilesTab_StepDefinitions.java:1]: not needed method

Comment (rokudaimme) [reply src/test/java/com/project/step_definitions/DeletedFilesTab_StepDefinitions.java:1]: rest is good!

**Review by rokudaimme — APPROVED** [review]

**Review by nannatalia — CHANGES_REQUESTED** [review]
there is already a File page in the master, with the name: FilesPage

**Comment by nannatalia** [comment]
> there is already a File page in the master, with the name: FilesPage
update the project, than copy-paste the webelements that are not there

**Review by nannatalia — APPROVED** [review]

**Pull request merged**